yt/loaders.py

```diff
@@ -151,8 +155,26 @@ def load_simulation(fn, simulation_type, find_outputs=False):
 # --- Loaders for generic ("stream") data ---
 
 
+def _sanitize_axis_order_args(
+    geometry: Union[str, Tuple[str, AxisOrder]], axis_order: Optional[AxisOrder]
+) -> Tuple[str, Optional[AxisOrder]]:
+    # this entire function should be removed at the end of its deprecation cycle
+    geometry_str: str
+    if isinstance(geometry, tuple):
```

Comment:
> since they would still see the deprecation warning, I also think it's OK not bothering to catch it.

that was my reasoning. But I'm okay with making the check more thorough if you think it'd improve clarity. Your call !

Reply:
Ok! No, no need to add it.